opentext-contact-us-page.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Opentext Contact Us</title>
+        <style>
+            container {
+                max-width: 1200px;
+            }
+            .row {
+                clear: float;
+            }
+            .column {
+                width: 25%;
+                float: right;
+            }
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            li a {
+                display: inline;
+            }
+            .col-6 {
+                width: 16%;
+                float: left;    
+            }
+            form {
+                clear: both;
+            }
+        </style>
+        <script>
+            console.log("Hello World!")
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <nav>
+                        <ul>
+                            <li><a href="#Contact">Contact</a></li>
+                            <li><a href="#Global">Global</a></li>
+                            <li><a href="#SignIn">Sign In</a></li>
+                            <li><a href="#Search">Search</a></li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="col-6">
+                    <ul>
+                        <li><a href="#Home">OpenText</a></li>
+                        <li><a href="#Products">Products & Solutions</a></li>
+                        <li><a href="#Industries">Industries</a></li>
+                        <li><a href="#Services">Services</a></li>
+                        <li><a href="#Support">Support</a></li>
+                        <li><a href="#Resources">Resources</a></li>
+                    </ul>
+                </div>        
+                <form action="/action-page.php">
+                    <label for="inquiry">Inquiry type</label><br>
+                    <select>
+                        <option value="" disabled selected hidden>-- Please select --</option>
+                        <option>General inquiries</option>
```

Comment:
Each option requires a "value", so that when the option is selected, that value is what's used in the form data.
Please see:
https://www.w3schools.com/tags/tag_select.asp